lib/libmuser.c

```diff
@@ -663,13 +589,6 @@ static struct transport_ops {
     int (*get_request)(lm_ctx_t*, struct vfio_user_header*, int *fds, int *nr_fds);
     ssize_t (*recv_fds)(lm_ctx_t*, void *buf, size_t size);
 } transports_ops[] = {
```

Comment:
are you planning to keep the transport indirection around?

Reply:
Since we only support a single transport it doesn't really make sense to keep it, but it doens't terribly complicate the code IMO, so I'd suggest to keep it.